### PR TITLE
refactor(edgeless): remove phasor package

### PR DESCRIPTION
**Pull request by regischen**
remove phasor package

### CODE REVIEW

File: tests/utils/actions/edgeless.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import '../declare-test-window.js';
...
 
-import type {
-  CssVariableName,
-  IPoint,
-  NoteBlockModel,
+import {
+  type CssVariableName,
+  type IPoint,
+  type NoteBlockModel,
+  Vec,
```
Comment (fourdim) [inline tests/utils/actions/edgeless.ts:11]: Remove this from tests.

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1514,7 +1299,7 @@ packages:
       '@types/node': 18.17.11
       playwright-core: 1.37.1
     optionalDependencies:
-      fsevents: 2.3.2
+      fsevents: registry.npmmirror.com/fsevents@2.3.2
```
Comment (fourdim) [inline pnpm-lock.yaml:1302]: Remove the npmmirror

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -28,100 +28,100 @@ importers:
     devDependencies:
       '@commitlint/cli':
         specifier: ^17.7.1
-        version: 17.7.1
+        version: registry.npmmirror.com/@commitlint/cli@17.7.1
```
Comment (fourdim) [inline pnpm-lock.yaml:31]: Remove the npmmirror.

**Review by fourdim — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml (hunk 3 above)
Comment (fourdim) [reply pnpm-lock.yaml:31]: https://pnpm.io/npmrc#https-proxy
https://pnpm.io/npmrc#http-proxy

Comment (regischen) [reply pnpm-lock.yaml]: why should we modify pnpm-lock.yaml manuallly? it's generated

Comment (fourdim) [reply pnpm-lock.yaml:31]: You should not modify it manually. What you need to do is to use the proxy to pull from npmjs rather than the mirror.

Comment (fourdim) [reply pnpm-lock.yaml:31]: For oversea users, the npmmirror field in the pnpm lock file will force the pnpm to request the package from the npmmirror. That will slow down their installation process (including CI), and may violate their goverment or company policies.

File: pnpm-lock.yaml [hunk 4]
```diff
@@ -162,6 +159,9 @@ importers:
       file-type:
         specifier: ^16.5.4
         version: 16.5.4
+      fractional-indexing:
+        specifier: ^3.2.0
+        version: registry.npmmirror.com/fractional-indexing@3.2.0
```
Comment (fourdim) [inline pnpm-lock.yaml:164]: still

File: pnpm-lock.yaml (hunk 3 above)
Comment (regischen) [reply pnpm-lock.yaml]: okay, it's because in my machine I use npm mirror. 
It would be fine that I use the official npm source.

**Review by fourdim — APPROVED** [review]

**Pull request merged**